db/migrate/20200422154616_rename_satellite_initial_value.rb

```diff
@@ -0,0 +1,19 @@
+class RenameSatelliteInitialValue < ActiveRecord::Migration[5.2]
+  def up
+    SourceType.where(:name => "satellite").each do |type|
+      type.schema["endpoint"]["fields"].each do |field|
+        field["initialValue"] = "satellite" if field["initialValue"] == "sattelite"
+      end
+      type.save!
+    end
+  end
+
+  def down
+    SourceType.where(:name => "satellite").each do |type|
+      type.schema["endpoint"]["fields"].each do |field|
+        field["initialValue"] = "sattelite" if field["initialValue"] == "satellite"
+      end
+      type.save!
+    end
+  end
```

Comment:
@lindgrenj6 These methods are largely identical.  Could you do a quickly DRY'ing up of this code by making a method that takes `old_name` and `new_name` parameters and the up/down methods just call that.

Reply:
Sure, made a `rename` method that does the dirty work.